lib/java/src/main/java/org/apache/thrift/ProcessFunction.java

```diff
@@ -8,86 +8,91 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public abstract class ProcessFunction<I, T extends TBase> {
-  private final String methodName;
+public abstract class ProcessFunction<I, T extends TBase, A extends TBase> {
+    private final String methodName;
```

Comment:
I believe this is the commit that breaks spotless check for java in CI (and thus stops running all cross tests): https://github.com/apache/thrift/actions/runs/8048152173/job/22320319518

@thomasbruggink can you please run `./gradlew :spotlessApply` and commit the format changes it made in a new PR? thank you.

Reply:
Yes, let me make a new PR today.

Reply:
PR: https://github.com/apache/thrift/pull/2945